### DISCUSSION TITLE
Live in the real world

### DISCUSSION THREAD

**Discussion by mholtz16** [Azure/azure-storage-fuse#1004] [Show and tell]
I wanted to say thanks to the developers and maintainers.  I am currently using blobfuse2 on 3 ubuntu servers connected to a 140TB container. These machines are service about 1500 files a minute and it is working like a champ!

**Comment by bhansalivikas** [comment]
@mholtz16: thanks a ton. Getting positive feedback is a rare event, as most users contact when there is some issue or query :)
Really appreciate your kind words and it boosts team's morale.
